neurom/apps/tests/test_morph_stats.py

```diff
@@ -143,6 +143,20 @@ def test_extract_stats_single_neuron():
             assert_almost_equal(res[k][kk], REF_OUT[k][kk], places=3)
 
 
+def test_extract_stats_single_neuron_path():
```

Comment:
Reuse the previous test and add a for loop top level on the argument passed to `extract_stats`
```python
path = SWC_PATH / 'Neuron.swc'
for nrn in (nm.load_neuron(path), path, str(path)):
```